HidSharp/HidDevice.cs

```diff
@@ -48,6 +48,8 @@ public override string GetFriendlyName()
             return GetProductName();
         }
 
+        public abstract void Seize();
```

Comment:
Might be a good idea to instead leave this out of HidDevice for now, because only MacOS supports it. We can instead add it to an ISeizable interface that MacHidDevice can inherit. If we end up getting it to work on all 3 platforms, we'll move it to HidDevice.

Reply:
Moved to ISeizable interface as requested